db/seeds/fill.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'factory_bot_rails'
+require 'faker'
+
+contributors_count = 100
+request_count = 200
+message_count = request_count * (contributors_count * 0.5).to_i
+message_time = Faker::Time.backward(days: 14)
+
+users = User.all
+
+# images = 10.times.map { URI(Faker::Avatar.image(size: '50x50', format: 'png', set: 'set5')) }
+
+FactoryBot.modify do
+  factory :contributor do
+    data_processing_consent { true }
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    note { Faker::Movies::HitchhikersGuideToTheGalaxy.quote }
+
+    # after(:build) do |contributor|
+    #   image = images.sample
+    #   contributor.avatar.attach(
+    #     io: image.open,
+    #     filename: File.basename(image.path)
+    #   )
+    # end
+  end
+end
+
+FactoryBot.modify do
+  factory :request do
+    title { Faker::Lorem.question }
+    text { Faker::Lorem.paragraph }
+    user { users.sample }
+  end
+end
+
+Rails.logger.debug 'Seeding requests..'
+requests = FactoryBot.create_list(:request, request_count)
+
+Rails.logger.debug 'Seeding contributors..'
+contributors = FactoryBot.create_list(:contributor, contributors_count)
+
+FactoryBot.modify do
+  factory :message do
+    created_at { message_time }
+    updated_at { message_time }
+    sender_type { 'Contributor' }
+    text { Faker::Lorem.paragraph }
+    unknown_content { false }
+    broadcasted { false }
+    sender { contributors.sample }
+    recipient { nil }
+    request { requests.sample }
+  end
+end
+
+Rails.logger.debug 'Seeding messages..'
+FactoryBot.create_list(:message, message_count)
```

Comment:
I tried this out a few weeks ago, before I saw Robert's comments about the code changing and the motive behind it. I also like the idea of seeding the db locally and restoring it to a production instance.

I mentioned this to @roschaefer already, but I wanted to review it. I ran into an issue when I ran the rake task because I had some of my own contributors already in the db, and the creation of new messages resulted in our after_create_commit being run:

```
after_create_commit :send_if_outbound
```

I killed all jobs as quickly as possible, but not before several messages went out to WhatsApp and Threema contributors, which are paid messages. I also commented out  the after_create_commit and re-ran the script, but ideally one wouldn't need to remember doing this.

@roschaefer do you have any experience with skipping callbacks? I saw there are a few ways to do it. This issue on factory_bot discussing one way that we could probably easily implement here See https://github.com/thoughtbot/factory_bot/issues/931#issuecomment-307542965

Other than that, I think it's really nice to have this. I would like to re-seed my db to test out the performance improvements I am working on at the moment

Reply:
I went ahead and added the code I was suggesting here. See 2d4015f2.

Reply:
Well it changes the class itself instead of the instance. But I think that won't have any side effects as we want this behavior for the whole seeding script. Seems to work. Fine for me